spellcheck.dic

```diff
@@ -0,0 +1,248 @@
+247
```

Comment:
We already have a `.spelling` file - please do not introduce a second dictionary.